src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence.mdx

```diff
@@ -274,69 +256,107 @@ Workflows will soon be replacing configurations. Learn more about our workflows
 
+  </Collapser>
 
-### Anomaly visibility settings [#visibility-settings]
+    <Collapser
+    className="freq-link"
+    id="jira"
+    title="Anomaly visibility settings"
+    >
+  
+  Anomalies are displayed in various New Relic activity streams and in the applied intelligence anomalies feed. You can customize what is displayed using the anomaly visibility settings (for example, hiding throughput anomalies on an activity stream but keeping them in the anomalies feed).
 
-Anomalies are displayed in various New Relic activity streams and in the applied intelligence anomalies feed. You can customize what is displayed using the anomaly visibility settings (for example, hiding throughput anomalies on an activity stream but keeping them in the anomalies feed).
+  To find these settings: from **Alerts & AI**, under **Anomaly detection**, click **Settings**.
 
-To find these settings: from **Alerts & AI**, under **Anomaly detection**, click **Settings**.
+  Notes on using these settings:
 
-Notes on using these settings:
+  * These settings are applied at the user level. Changes you make won’t affect others users in your organization.
+  * Regardless of these settings, the anomalies are still reported and available for [NRQL querying](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language).
 
-* These settings are applied at the user level. Changes you make won’t affect others users in your organization.
-* Regardless of these settings, the anomalies are still reported and available for [NRQL querying](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language).
+  Details on these UI sections:
 
-Details on these UI sections:
+  * **AI overview and anomalies tab**: Use the AI overview and anomalies tab setting to hide anomalies from the AI overview and anomalies tab setting. Please note you also can use filters specific to these views as well.
+  * **Global activity stream**: Use the global activity stream section to customize what anomalies are shown in the various New Relic activity streams, including the New Relic homepage, APM Summary, and Lookout.
+  * **Anomaly types**: Use the check boxes here to hide specific types of anomalies. For example, uncheck **Web throughput** and **Non-web throughput** anomalies to hide these types of anomalies from both the activity streams and the AI overview and anomalies tab. (Note they are still reported and available for querying.)
```

Comment:
I don't think you need parentheses for the last sentence if it is going to be separate from the predeecing one.